client/js/components/assessments/universal_input.jsx

```diff
@@ -35,43 +35,47 @@ export default class UniversalInput extends React.Component{
   }
 
   wasSelected(id){
-    if( this.props.response ){
+    if(this.props.response){
       return this.props.response.indexOf(id) > -1;
     } else {
       return null;
     }
   }
 
```

Comment:
Previously this functionality existed in the map function that rendered radio buttons, I pulled them out into their own methods to reuse them for checkbox